lib/rbUtil.js

```diff
@@ -108,31 +108,24 @@ rbUtil.tidFromDate = function tidFromDate(date) {
         throw new Error('Invalid date');
     }
     // Create a new, deterministic timestamp
-    return uuid.v1({
-        node: [0x01, 0x23, 0x45, 0x67, 0x89, 0xab],
-        clockseq: 0x1234,
-        msecs: +date,
-        nsecs: 0
-    });
+    return uuid.TimeUuid.fromDate(date,
+        0,
+        new Buffer([0x01, 0x23, 0x45, 0x67, 0x89, 0xab]),
+        new Buffer([0x12, 0x34])).toString();
 };
 
-var uuidRe = /^[0-9a-fA-F]{8}-[0-9a-fA-F]{4}-[0-9a-fA-F]{4}-[0-9a-fA-F]{4}-[0-9a-fA-F]{12}$/;
 /**
  * Check if a string is a valid timeuuid
  */
 rbUtil.isTimeUUID = function (s) {
-    return uuidRe.test(s);
+    return uuid.TimeUuid.test(s);
```

Comment:
Actually, let's check whether `isTimeUUID()` is even used. IMO, we've eliminated it. IF so, we can remove it entirely.